{{cookiecutter.repository_name}}/setup.py

```diff
@@ -32,9 +32,9 @@
     'watchdog>=0.8.3',
 
     # docs
-    'm2r>=0.2.0',
-    'Sphinx>=1.7.1',
-    'sphinx_rtd_theme>=0.2.4',
+    'm2r>=0.2.0,<0.3',
```

Comment:
have we experienced trouble with these max versions? possible to describe the problem somewhere, either in an issue/email/slack?

Reply:
A while back, a bunch of repositories - Copulas, SDV, etc. - broke because of a version issue here. I don't recall the exact problem but Carles and I decided to cap the Sphinx versions, I'll see if I can find the details.

Reply:
I think that these versions should be reviewed. What actually provoked the errors was the release of Sphinx 3.0.0, which changed the API and made the m2r addon crash.

You can see the corresponding m2r issue here: https://github.com/miyakogi/m2r/issues/51

So, rather than capping m2r, I think that we should cap Sphinx to < 3 until this issue is fixed on m2r.